dist/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+    <title>ToDoApp</title>
+<script defer src="main.js"></script></head>
+<body>
+    
+    <ul class="container">
+        <li>
+            <h3 class="title">barna To Do</h3>
+        </li>
+        <li> 
+            <form action="" class="insert">
+                 <input type="text" name="insert" id="insert" placeholder=" Add to your list..." required>
```

Comment:
- [ ] There is still an issue in adding a to-do item as stated in the previous review. Please fix it.